apis/duck/typed_test.go

```diff
@@ -62,19 +62,18 @@ func TestSimpleList(t *testing.T) {
 		},
 	})
 
-	stopCh := make(chan struct{})
-	defer close(stopCh)
+	ctx := context.Background()
```

Comment:
This should get a `WithCancel` and then a `defer cancel()` to represent the same behavior.